scripts/installers/Dockerfile.ngramtools

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this script to install KenLM, OpenSeq2Seq decoder, Flashlight decoder, OpenGRM tool in contaner 
+
+# How to use? Build it from NeMo root folder:
+# 1. git clone https://github.com/NVIDIA/NeMo.git && cd NeMo
+# 2. DOCKER_BUILDKIT=1 docker build -t nemo:23.01.1 -f ./scripts/installers/Dockerfile.ngramtools .
+
+from nvcr.io/nvidia/nemo:23.01
```

Comment:
How could I break in the future if it is hardcoded?

Reply:
Because you are copying scripts from main but using older Nemo container. It's fine but we need to update this base container when new versions come out.

Reply:
I see, as bought required files explicitly listed it will be easy to handle